src/dipdup/config/__init__.py

```diff
@@ -414,10 +414,10 @@ class HasuraConfig:
     admin_secret: str | None = Field(default=None, repr=False)
     create_source: bool = False
     source: str = 'default'
-    select_limit: int = 1000
+    select_limit: int = 10_000
     allow_aggregations: bool = True
     allow_inconsistent_metadata: bool = False
-    camel_case: bool = False
+    camel_case: bool = True
```

Comment:
false?

Reply:
Currently there's an inconsistency. In _config class_ default is False. In environment configs provided with project template default is True. Thus, _de-facto_ default is True.  

Changing default in config class would be a breaking change. I'll double check env vars in our deployments after merging this.